src/emulator/auth/state.ts

```diff
@@ -881,6 +907,19 @@ export interface PhoneVerificationRecord {
   sessionInfo: string;
 }
 
+export enum UsageMode {
+  DEFAULT = "DEFAULT",
+  PASSTHROUGH = "PASSTHROUGH",
+
+  // Should never be used
+  USAGE_MODE_UNSPECIFIED = "USAGE_MODE_UNSPECIFIED",
```

Comment:
nit: please move this value to the top (to follow the proto ordering of enum values)